Dockerfile

```diff
@@ -16,7 +16,7 @@ ARG VERSION
 # suggested that you use an email address here that is specific to the
 # person or group that maintains this repository; for example:
 # LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
-LABEL org.opencontainers.image.authors="github@cisa.dhs.gov"
```

Comment:
Now that you've updated this, you can remove the comment just about the email address.  Be careful not to remove the comment about the annotation keys.